lib/bot.js

```diff
@@ -185,8 +187,16 @@ class Bot {
         return;
       }
 
+      const currentChannelUsernames = slackChannel.members.map( member =>
+        slackChannel._client.users[member].name
```

Comment:
Maybe you could use `this.slack.getUserByID(member)` here instead? I'm a bit worried about using node-slack-client's private API since it's been rewritten completely [here](https://github.com/l12s/node-slack-client), which means the underlying properties etc. might have changed.